hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/ratis/SCMRatisServer.java

```diff
@@ -479,4 +425,166 @@ private void setServerRole(RaftPeerRole currentRole,
       this.roleCheckLock.writeLock().unlock();
     }
   }
+
+  /**
+   * Configuration used by SCM Ratis Server.
+   */
+  @ConfigGroup(prefix = "ozone.scm.ratis")
```

Comment:
Is it possible to make these RATIS related configuration reusable among DN,SCM and OM? 
Just different prefix like, ozone.om.ratis, ozone.scm.ratis, ozone.dn.ratis as prefix?

Reply:
Yea it looks like we currently have ozone.om.ratis, ozone.scm.ratis, hdds.datanode.ratis, dfs.container.ratis and even dfs.ratis.

If we decide to refactor all config into Java-based configuration, we shall consider refactoring all config to be like ozone.ratis.*

Then we can take a look at reusable configuration among OM, SCM and container. I create Jira to track: https://issues.apache.org/jira/browse/HDDS-3577